src/wlapplication_messages.cc

```diff
@@ -19,177 +19,236 @@
+    false},
+   /// Paths
+   {" ", "datadir", _("DIRNAME"), _("Use specified directory for the widelands\ndata files"),
+    false},
+   {"", "homedir", _("DIRNAME"),
+    _("Use specified directory for widelands config\nfiles, savegames and replays\nDefault is ") +
+       kDefaultHomedir,
+    false},
+   {"", "localedir", _("DIRNAME"), _("Use specified directory for the widelands\nlocale files"),
+    false},
+   {" ", "language", _("[de_DE|sv_SE|...]"), _("The locale to use"), false},
+   /// Game setup
+   {" ", "new_game_from_template", _("FILENAME"),
+    _("Directly create a new singleplayer game\nconfigured in the given file. An example can\nbe "
+      "found in `data/templates/new_game_template`"),
+    false},
+   {"", "scenario", _("FILENAME"), _("Directly starts the map FILENAME as scenario map"), false},
+   {"", "loadgame", _("FILENAME"), _("Directly loads the savegame FILENAME"), false},
+   {"", "replay", _("FILENAME"), _("Directly loads the replay FILENAME"), false},
+   {"", "script", _("FILENAME"),
+    _("Run the given Lua script after initialization.\nOnly valid with --scenario, --loadgame, or "
+      "--editor"),
+    false},
+   {"", "editor", "",
+    _("Directly starts the Widelands editor.\nYou can add a =FILENAME to directly load\nthe map "
+      "FILENAME in editor"),
+    false},
+   /// Misc
+   {" ", "nosound", "", _("Starts the game with sound disabled"), false},
+   {" ", "fail-on-lua-error", "", _("Force Widelands to crash when a Lua error occurs"), false},
+   {"", "ai_training", "",
+    _("Enables AI training mode. See\nhttps://www.widelands.org/wiki/Ai%20Training/\nfor a full "
+      "description of the AI training logic"),
+    false},
+   {"", "auto_speed", "",
+    _("In multiplayer games only, this will keep\nadjusting the game speed "
+      "automatically,\ndepending on FPS. Useful in conjunction with\n--ai_training"),
+    false},
+   /// Saving options
+   {"", "autosave", _("n"), _("Automatically save each n minutes"), false},
+   {"", "rolling_autosave", _("n"), _("Use n files for rolling autosaves"), true},
+   {"", "nozip", "", _("Do not save files as binary zip archives"), false},
+   {"", "display_replay_filenames", _("[true*|false]"), _("Show filenames in replay screen"), true},
+   {"", "editor_player_menu_warn_too_many_players", _("[true*|false]"),
+    _("Enable verbose debug messages"), true},
+   /// Game options
+   {"", "auto_roadbuild_mode", _("[true*|false]"), _("Start building road after placing a flag"),
+    true},
+   {"", "display_flags", _("[...]"), _("Display flags to set for new games"), true},
+#if 0  // TODO(matthiakl): Re-add training wheels code after v1.0
+	{"",
+	 "training_wheels",
+	 _("[true*|false]"),
+	 "",
+	 true
+	},
+#endif
+   {"", "edge_scrolling",
+    /** TRANSLATORS: You may translate true/false, also as on/off or yes/no, but */
+    /** TRANSLATORS: it HAS TO BE CONSISTENT with the translation in the widelands textdomain. */
```

Comment:
This hint should be repeated for every string that contains "true|false". Also, some strings have a '*' character which should be explained in a translators comment on each such string.

Reply:
I placed it on the first occurrence of each string. From a look into the .pot after a local buildcat.py this is sufficient. 
The comment about the * got lost somehow, it is now back.